src/app.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+import Main from './components/main';
+
+import './components/general.scss';
+
+const Wrapper = (
+    <>
+        <Main title="Title" />
+    </>
```

Comment:
you don't need here react fragment `<>...</>` (because you have only one component)